package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Jul 23 16:00:07 UTC 2015 - mfilka@suse.com
+
+- bnc#916013
+  - enables IPv6 forwarding
```

Comment:
I guess that is not the actual fix, as the IPv6 forwarding checkbox existed even before this change. Can you please describe what is being fixed from the user's point of view?